src/Components/List/List.jsx

```diff
@@ -130,12 +147,29 @@ export default function List({ items, token }) {
                     onChange={() => purchaseItem(item, token)}
                     checked={checked}
                     disabled={checked}
+                    aria-label={`Mark as purchased`}
                   />
                   <label htmlFor={item.name}>{item.name}</label>
+                  <button
+                    className="deleteItem"
+                    onClick={() => setDeleteItemName(item.name)}
+                    aria-label={`Delete ${item.name}`}
```

Comment:
Nice job remembering the arias!

Reply:
I  agree, great job on creating accessible code 👍